python/graphstorm/gconstruct/construct_graph.py

```diff
@@ -591,6 +591,16 @@ def verify_confs(confs):
             "The config file does not have a 'version' entry. Assuming gconstruct-v0.1")
     ntypes = {conf['node_type'] for conf in confs["nodes"]}
     etypes = [conf['relation'] for conf in confs["edges"]]
+    # Adjust input to DGL requirement if it is a honogeneous graph
+    if len(ntypes) == 1 and len(etypes) == 1 and not rev_edges:
```

Comment:
There can be multiple blocks under confs["nodes"] having the same ntype.
See https://github.com/awslabs/graphstorm/blob/main/tests/end2end-tests/data_process/data_gen.py#L155-L193 as an example.